### DISCUSSION TITLE
How to bypass ssl certificate validation

### DISCUSSION THREAD

**Discussion by hermanmaleiane** [CrowdStrike/falconpy#673] [Q&A] [answered]
Hi team,

How can i bypass certificate validation to false.
Im getting this error:

{'status_code': 500, 'headers': {}, 'body': {'errors': [{'message': "HTTPSConnectionPool(host='api.crowdstrike.com', port=443): Max retries exceeded with url: /devices/queries/devices/v1?filter=hostname%3A%27WDSI249%27 (Caused by SSLError(SSLCertVerificationErrr(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:997)')))"}], 'resources': []}}

I want to do something like this:

api = TheHiveApi('https:xxxxxxxxxxx', 'xxxxxxx',cert=False)

Thanks in advance

**Comment by jshcodes** [comment, marked as answer]
Hi @hermanmaleiane -

If you are using a FalconPy class, you can pass `ssl_verify` and set it to False, examples can found [here](https://www.falconpy.io/Usage/Environment-Configuration.html#ssl-verify).

If you're making this call via requests directly, you should review the content [here](https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification).  The `verify` keyword can be a little strange, and is not always a boolean.
